app/src/main/res/layout/profile_edit_fragment.xml

```diff
@@ -37,8 +37,8 @@
         android:id="@+id/profile_edit_image"
         android:layout_width="90dp"
         android:layout_height="90dp"
-        android:layout_marginStart="28dp"
-        android:layout_marginTop="28dp"
+        android:layout_marginStart="@dimen/profile_edit_image_margin_start"
+        android:layout_marginTop="@dimen/profile_edit_image_margin_top"
```

Comment:
Because these values are are same for landscape as well as portrait please remove it and instead directly write `28dp`